api/app/fire_behaviour/cffdrs.py

```diff
@@ -545,12 +545,25 @@ def fine_fuel_moisture_code(ffmc: float, temperature: float, relative_humidity:
 
     if ffmc is None:
         ffmc = NULL
```

Comment:
```suggestion
        logger.error("Failed to calculate FFMC; initial FFMC is required.")
        return None
```

Reply:
I guess we can't calculate any of the indices if any provided weather input is None/NULL. Maybe we should just do an `any` check to see if any of them are `None` and avoid calling the R library at all? Either way works I suppose

Reply:
These R functions are so inconsistent. Some will throw an error if not given an initial value for the FWI of interest, others simply return None (or whatever the equivalent is). Some functions allow NULL for a weather variable and others don't. 

I'll add this change but the leave the rest as is. Hopefully we'll get rid of this code in the next few months.